src/inc/quic_tls.h

```diff
@@ -417,7 +417,7 @@ CxPlatTlsParamGet(
     _In_ CXPLAT_TLS* TlsContext,
     _In_ uint32_t Param,
     _Inout_ uint32_t* BufferLength,
-    _Inout_updates_bytes_opt_(*BufferLength)
+    _Out_writes_bytes_opt_(*BufferLength)
```

Comment:
Let's just not change any of these annotations for now and leave fixing these up for that other issue.